pkg/types/types.go

```diff
@@ -66,6 +66,9 @@ type NetConf struct {
 	IPAM         struct {
 		Type string `json:"type,omitempty"`
 	} `json:"ipam,omitempty"`
+	IPAM6 struct {
+		Type string `json:"type,omitempty"`
+	} `json:"ipam6,omitempty"`
```

Comment:
Seems OK, but I wonder if the same ipam plugin might want to coordinate IPv4 and IPv6 addresses for a container, and having separate blocks precludes that.